src/main/java/com/kelinci/mainapp/api/UserController.java

```diff
@@ -12,11 +12,14 @@
 import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 @RestController
 public class UserController {
 
     private final List<OurUser> listOfUsers = new ArrayList<>();
+    private final List<RegisteredUserResponse> registeredUserResponses = new ArrayList<>();
```

Comment:
A to po co?

Reply:
a tak, chyba na początku jak pisałem to mi sugerowało wyrzucenie do klasy. Usunąłem.